internals/scripts/setup.js

```diff
@@ -1,151 +0,0 @@
-#!/usr/bin/env node
```

Comment:
I dont think you should remove this. We will need this later

Reply:
Please leave this, you also left reference to this in the package.json

Reply:
If the goal is to change what the default clean command does, then change the name of the current clean command in the package.json and add your new command. This one is actually very useful it cleans out the default project.